### PR TITLE
refactored the skill description

### PR DESCRIPTION
**Pull request by MuhammadKurdi-CS**
refactored skill description

### CODE REVIEW

File: src/components/Contribute/Skill/index.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { getGitHubUsername } from '../../../utils/github';
 import { useSession } from 'next-auth/react';
 import YamlCodeModal from '../../YamlCodeModal';
 import { AttributionData, SchemaVersion, SkillYamlData } from '@/types';
-import SkillDescription from './SkillDescription';
+import SkillDescription from './SkillDescription/SkillDescriptionContent';
```
Comment (aevo98765) [inline src/components/Contribute/Skill/index.tsx:19]: This is not the correct component to import. You have imported just the content. The 'wrapper' component SkillDescription is what should be imported here. 

If you run `npm run dev` you can see that the Expandable section is not being rendered.

Compare the your branches UI to the main branch UI and this should make sense.

**Review by aevo98765 — CHANGES_REQUESTED** [review]

File: src/components/Contribute/Skill/index.tsx (hunk 1 above)
Comment (MuhammadKurdi-CS) [reply src/components/Contribute/Skill/index.tsx]: thank you for bringing this to attention, the rendering issue has been resolved.

**Review by vishnoianil — APPROVED** [review]
LGTM

**Review by aevo98765 — APPROVED** [review]

**Comment by aevo98765** [comment]
Thanks for this one @MuhammadKurdi-CS

**Pull request merged**